hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/committer_architecture.md

```diff
@@ -1312,6 +1312,16 @@ On `close()`, summary data would be written to the file
 `/results/latest/__magic/job400_1/task_01_01/latest.orc.lzo.pending`.
 This would contain the upload ID and all the parts and etags of uploaded data.
 
+A marker file is also created, so that code which verifies that a newly created file
+exists does not fail.
+1. These marker files are zero bytes long.
+1. They declare the full length of the final file in the HTTP header
+   `x-hadoop-s3a-magic-marker`.
```

Comment:
you mean `x-hadoop-s3a-magic-data-length` here?

Reply:
nicely spotted. will fix